.changeset/tsc.md

```diff
@@ -0,0 +1,32 @@
+---
+"@biomejs/js-api": fix
```

Comment:
```suggestion
"@biomejs/js-api": major 
```
  Isn't this a breaking change? We're are drastically changing how the JavaScript library is used.

Reply:
Ok, I updated the suggestion so it's a minor, because we're adding a new way to use the APIs.